src/hooks/useColorGenerator/__stories__/useColorGenerator.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+import type {Meta, StoryObj} from '@storybook/react';
+
+import {block} from '../../../components/utils/cn';
+import {randomString} from '../../../components/utils/common';
+
+import {ColoredAvatar} from './ColoredAvatar';
+
+import './ColorGenerator.scss';
+
+const b = block('color-generator');
+
+const meta: Meta = {
+    title: 'Hooks/useColorGenerator',
+    argTypes: {
+        withText: {
+            control: 'boolean',
+        },
+    },
+};
+
+export default meta;
+
+type Story = StoryObj<typeof ColoredAvatar>;
+
+const views = ['-', 'light', 'medium', 'heavy'] as const;
```

Comment:
I think we can remove first header row completely, so it will be three rows, one row for each intensity. Will be much cleaner